environment/vagrant/training/vb-multi-vm/TestIndyClusterSetup.md

```diff
@@ -1,14 +1,16 @@
 # Setting Up a Test Indy Network in VMs
 
+**WARNING:** Here use deprecated client. Please, use [new client from SDK](https://github.com/hyperledger/indy-sdk/tree/master/cli)
```

Comment:
I would write it as follows:
**WARNING:** This guideline assumes using of a deprecated CLI and deprecated Getting Started Guide.
Please have a look at [new client from SDK](https://github.com/hyperledger/indy-sdk/tree/master/cli) and [new Getting Started Guide](...) instead.